Base/QTCore/qSlicerExtensionsManagerModel.h

```diff
@@ -142,7 +141,7 @@ class Q_SLICER_BASE_QTCORE_EXPORT qSlicerExtensionsManagerModel : public QObject
   /// \sa installExtension, isExtensionInstalled, installedExtensions, extensionInstalled
   int numberOfInstalledExtensions()const;
 
-  /// \brief Return names of all installed extensions
+  /// \brief Return names of all installed extensions sorted in alphapeicall order.
```

Comment:
```suggestion
  /// \brief Return names of all installed extensions sorted in alphabetical order.
```

Reply:
Re: replace the extensions manager model with the content of #6417

Exactly I started on this.